projects/ngrx.io/content/guide/effects/lifecycle.md

```diff
@@ -36,6 +38,40 @@ export class LogEffects {
 }
 </code-example>
 
+### Resubscribe on Error
+
+Starting from NgRx V8 when error happens in the effect's main stream it is
+reported using Angular's `ErrorHandler` and the source effect is 
+**automatically** resubscribe (instead of completeling), so it continues to 
+listen to any new dispatched Actions.
+
+Generally, errors should be handled by users and operators such as `mapToAction`
+should make it easier to do. However, for the cases where errors were missed, 
+this new behavior adds additional safety net.
+
+In some cases where particular RxJS operators are used the new behavior might
+produce unexpected results. For example, if the `startWith` operator is within 
+effect's pipe then it will be triggered again.
+
+To remove resubscriptions add `{resubscribeOnError: false}` to `createEffect` 
+metadata (second argument).
+
+<code-example header="disable-resubscribe.effects.ts">
+import { Injectable } from '@angular/core';
+import { Actions, Effect, ofType } from '@ngrx/effects';
+import { tap } from 'rxjs/operators';
+
+@Injectable()
+export class ErrorFreeEffects {
```

Comment:
I think we need a different example here. This infinite loop has bitten us enough times.

Reply:
Provided what I hope is a better example